js/components/closed-caption-multi-audio-menu/closedCaptionMultiAudioMenu.js

```diff
@@ -42,13 +43,26 @@ var ClosedCaptionMultiAudioMenu = React.createClass({
    * when clicking on an item from an audio list, set the corresponding audio value
    * @param id {string} - id of clicked element
    */
-  handleMultiAudioClick: function(id) {
-    if (this.props.controller && typeof this.props.controller.setCurrentAudio === 'function') {
-      this.props.controller.setCurrentAudio(id);
+  handleMultiAudioClick: function (id) {
```

Comment:
Do we need a new unit test or update a unit test to test this change in handleMultiAudioClick?

Reply:
Yeah, the new tests are [here](https://github.com/ooyala/html5-skin/pull/1044/files#diff-3c346cedeb48ad57627bb9cf6b8b3ca6R94) and [here](https://github.com/ooyala/html5-skin/pull/1044/files#diff-63a4cbb7eb24c1dc5f84f0a7439d0d34R726)